fe/fe-core/src/main/java/org/apache/doris/analysis/Analyzer.java

```diff
@@ -17,6 +17,14 @@
 
 package org.apache.doris.analysis;
 
+import com.google.common.base.Joiner;
```

Comment:
Reorder the import , see
http://doris.incubator.apache.org/master/zh-CN/developer-guide/fe-eclipse-dev.html#import-%E9%A1%BA%E5%BA%8F